src/java_bytecode/jar_file.h

```diff
@@ -69,9 +98,9 @@ class jar_poolt:public messaget
   }
 
 protected:
+  /// A map from filesystem paths to jar_filet objects
   typedef std::map<std::string, jar_filet> file_mapt;
   file_mapt file_map;
-  std::string java_cp_include_files;
```

Comment:
Indeed, this field was not used. Maybe because `operator()` already receives an instance of `java_class_limitt`?